common/envHandler.js

```diff
@@ -33,6 +33,36 @@ envHandler.get = function (envName, cb) {
   );
 };
 
+envHandler.post = function (envName, envValue, cb) {
+  var createCmd = util.format('sudo echo \'%s="%s"\' >> %s',
```

Comment:
where is the validation for each of the inputs. its not there in the route handler or here. at least check if the values are not null

Reply:
why is there no check here. we cannot have a null env even though the value can be null. i dont think we're thinking this through. this sort of code creates more problems than it solves because of lack of resiliency